docs/getting-started-guides/libvirt-coreos.md

```diff
@@ -117,6 +117,26 @@ setfacl -m g:kvm:--x ~
 
 By default, the libvirt-coreos setup will create a single Kubernetes master and 3 Kubernetes nodes. Because the VM drives use Copy-on-Write and because of memory ballooning and KSM, there is a lot of resource over-allocation.
 
+There are two major ways of how-to run and configure the libvirt-based Kubernetes cluster on CoreOS instances:
+
+#### Automated setup
+
+To start and setup your local CoreOS-Kubernetes cluster the following command might be easily runned:
```

Comment:
To start and setup your local CoreOS-Kubernetes cluster the following command might be easily runned:

To set up and run your Kubernetes cluster on a local CoreOS machine, download `kubernetes.tar.gz` using the script at `http://get.k8s.io` with the `KUBERNETES_PROVIDER` environment variable set to `libvirt-coreos`: